extensions/amp-recaptcha-input/0.1/amp-recaptcha-service.js

```diff
@@ -85,13 +97,15 @@ export class AmpRecaptchaService {
   /**
    * Function to register as a dependant of the AmpRecaptcha serivce.
    * Used to create/destroy recaptcha boostrap iframe.
-   * @param {!Element} element
+   * @param {string} sitekey
    * @return {Promise}
    */
-  register(element) {
+  register(sitekey) {
     this.registeredElementCount_++;
     if (!this.iframeLoadPromise_) {
-      this.initialize_(element);
+      return this.initialize_(sitekey).then(() => {
```

Comment:
nit: what do you think about having `this.initialize_` return a iframeLoadPromise. Then we can have 
```
if (!this.iframeLoadPromise_) {
  this.iframeLoadPromise_ = this.initialize_();
}
return this.iframeLoadPromise_
```
I feel that's easier to understand. what do you think?